docs/Auth/2. Web/1.overview.md

```diff
@@ -0,0 +1,23 @@
+# Overview
+
+## Introduction
+
+JSON Web Token (JWT) is used for web authentication to securely transmit information between a client and a server as a JSON object. When a user logs in, the server validates the credentials and returns a signed JWT to the user's browser. This JWT contains information about the user's identity. AI DIAL, allows to use various Identity Service Providers (IDPs) to manage user identities.
```

Comment:
I would replace `When a user logs in, the server validates the credentials and returns a signed JWT to the user's browser` to `When a user logs in, the IDP validates the credentials and returns a signed JWT to DIAL chat back-end`